openshift/sandboxed-containers-operator
# Patch series to implement the missing functionality of kata-monitor

**- Description of the problem which is fixed/What is the use case**
Fixes: #[KATA-1200](https://issues.redhat.com/browse/KATA-1200)
Fixes: #[KATA-1205](https://issues.redhat.com/browse/KATA-1205)

**- What I did**
Created custom SCC for kata-monitor with minimum required privileges.
This is still using selinux `spc_t` type which will be removed in future patches.
Also added required rbac, sa, roles and namespace labels for kata-monitor

**- How to verify it**
Deploy the operator and check for scc and serviceaccount in the operator namespace

**- Description for the changelog**

Add support for automatic configuration of required kata-monitor settings via operator
